### PR TITLE
Add DITA Bootstrap version 5.3.2

### PR DESCRIPTION
**Pull request by infotexture**
Update DITA Bootstrap plug-in to version [5.3.2](https://github.com/infotexture/dita-bootstrap/releases/tag/5.3.2).

### CODE REVIEW

File: net.infotexture.dita-bootstrap.json [hunk 1]
```diff
@@ -126,5 +126,21 @@
...
+    "description": "HTML5 output with a basic Bootstrap template",
+    "keywords": ["HTML", "HTML5", "Bootstrap", "Bootstrap 5", "Responsive"],
+    "homepage": "https://infotexture.github.io/dita-bootstrap",
+    "vers": "5.3.2",
+    "license": "Apache-2.0",
+    "deps": [
+      {
+        "name": "org.dita.base",
+        "req": ">=3.7.0"
+      }
```
Comment (jason-fox) [inline net.infotexture.dita-bootstrap.json:141]: There is a missing dependency here:

```suggestion
      },
      {
        "name": "fox.jason.extend.css",
        "req": ">=1.0.0"
      }

```

Comment (infotexture) [reply net.infotexture.dita-bootstrap.json:141]: Ah, good catch. 🙇 

That's apparently been missing from earlier versions as well. 

We should update those entries as well.

**Comment by infotexture** [comment]
@jason-fox Also added for 5.3 & 5.3.1. 

Should be good to go now.

**Review by jason-fox — APPROVED** [review]
LGTM

**Pull request merged**